sn_client/src/files/api.rs

```diff
@@ -128,28 +139,65 @@ impl FilesApi {
         Ok(bytes)
     }
 
-    /// Tries to chunk the file, returning `(head_address, file_size, chunk_names)`
+    /// Tries to chunk the file, returning `(head_address, data_map_chunk, file_size, chunk_names)`
```

Comment:
maybe shall use this `head_address` consistantly for those `file xorname` ? (or even use `file_head_address` to be more explicit ?)

also I see there is using `data_map chunk` and `first_chunk`, which seems refer to the same thing and shall be consistant as well ?